bin/build-plugin-zip.sh

```diff
@@ -77,10 +77,17 @@ fi
 status "Gathering PHP dependencies... 🐿️"
 composer install --no-dev
 
-# Run the build.
-status "Generating build... 👷‍♀️"
-npm run build
-npm run docs
```

Comment:
Should docs still be built as a part of this? This doesn't get added back below.

Reply:
It was redundant because its part of the `npm run pre-release` script done earlier in the release process.